test/controllers/records_controller_test.rb

```diff
@@ -5,7 +5,7 @@ class RecordsControllerTest < ActionController::TestCase
   setup do
     collection_fixtures('bundles', 'records', 'vendors', 'products', 'product_tests', 'tasks', 'users', 'measures', 'roles')
     @vendor = Vendor.find(EHR1)
-    @first_product = @vendor.products.where(name: 'Vendor 1 Product 1').find('4f57a88a1d41c851eb000004')
+    @first_product = @vendor.products.where(name: 'Vendor 1 Product 1').by_updated_at.find('4f57a88a1d41c851eb000004')
```

Comment:
I'm not sure the by_updated_at is necessary here either since find only returns 1 object, ditto on all the other finds in the controller tests.